scripts/token-mold.js

```diff
@@ -1285,8 +1278,11 @@ class TokenMoldForm extends FormApplication {
     };
 
     let barAttributes = [];
-    for (let type in game.system.model.Actor) {
-      let newAttributes = getAttributes(game.system.model.Actor[type]).map(
+
+    const types = Actor.implementation.TYPES;
+    const shells = types.map(t => new Actor.implementation({ name: t, type: t }));
+    for (const actorElement of shells) {
+      let newAttributes = getAttributes(actorElement.toObject().system).map(
```

Comment:
Do we need toObject() here?  What type is actorElement? A JSON string?

Also, you can get away with const here.  newAttributes isn't changing in this loop once established, and it'll get overwritten for the next iteration anyway (consts are scoped to the block or iteration).

Reply:
Modified this to create `map[type] = Actor`. Iteration is the same, but the 'npc' look up later on is faster.

The `toObject()` was copied from the example code here: https://foundryvtt.com/article/active-effects/

I assume it's required (or optimal).